cypress/fixtures/members.ts

```diff
@@ -1,3 +1,5 @@
+import { MemberType } from '@graasp/sdk';
+
 export const MEMBERS = {
```

Comment:
You could enforce the type:
```suggestion
export const MEMBERS: { [name: string]: Member } = {
```